packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWModal/CWModal.scss

```diff
@@ -2,7 +2,7 @@
 
 .MuiModal-root {
   position: fixed;
-  z-index: 80;
+  z-index: 80000;
```

Comment:
Recently we had some z-index issues, i suggest we start using global variable for z-index values

Reply:
Why do we want popovers above the modals? Are the modals by design the top of the hierarchy? @mzparacha

Reply:
sometimes we want to use popover elements inside the modal, like a dropdown or tooltip